### PR TITLE
Enable focus mode and issue since column for non operator users

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
Enable focus mode and issue since column for non operator users.
For more details about this mode, see original PR: https://github.com/gardener/dashboard/pull/1235

Users can now enable this operator feature on the Settings page:
Screenshot 2023-09-18 at 10 49 41

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user
Enable focus mode for regular project lists. This feature used to be exclusive for operators. It can now be used by regular users on all cluster lists. This feature enables users to get a static overview of clusters with issues. In order to use this feature, you need to enable it on the `Settings` page.
```

### CODE REVIEW

**Review by petersutter — APPROVED** [review]
/lgtm

File: frontend/src/views/GShootList.vue [hunk 1]
```diff
@@ -704,6 +705,9 @@ export default {
       const items = this.sortItems(this.items, this.sortByInternal)
       return filter(items, item => this.searchItems(this.debouncedShootSearch, toRaw(item)))
     },
+    showOperatorFeatures () {
```
Comment (petersutter) [inline frontend/src/views/GShootList.vue:708]: rename to `issueSinceColumnVisible` / ``issueSinceColumnHidden`?

**Review by holgerkoser — APPROVED** [review]
/lgtm

File: frontend/src/views/GShootList.vue (hunk 1 above)
Comment (grolu) [reply frontend/src/views/GShootList.vue]: renamed to `issueSinceColumnVisible`

**Review by petersutter — APPROVED** [review]
/lgtm

**Pull request merged**